### PR TITLE
Feature: Redid set as background flyout

### LINKED ISSUE #7834: Add set as Lockscreen button to toolbar when image files are selected
### What's the Problem?

_No response_

### Solution/Idea

Adding a toolbar button will increase ease of use when setting an image as lockscreen.

### Alternatives

NA

### Priorities

_No response_

### Files Version

_No response_

### Windows Version

_No response_

### Comments

Related: [https://github.com/files-community/Files/issues/7769#issuecomment-1016431345](https://github.com/files-community/Files/issues/7769#issuecomment-1016431345)

### PR DESCRIPTION
**Pull request by puppetsw**
**Resolved / Related Issues**
Items resolved / related issues by this PR.
- Closes #7834

**Details of Changes**
Add details of changes here.
- Added set as flyout to toolbar.
- Set as desktop slideshow shows when multiple images as selected.

**Validation**
How did you test these changes?
- [x] Built and ran the app
- [ ] Tested the changes for accessibility

**Screenshots (optional)**
Screenshot 2022-08-20 212506
Screenshot 2022-08-20 212449

### CODE REVIEW

File: builds/azure-pipelines-release.yml [hunk 1]
```diff
@@ -32,7 +32,7 @@ variables:
 - name: solution
   value: '**/*.sln'
 - name: buildPlatform
-  value: 'x86|x64|arm64'
+  value: 'x64|arm64'
```
Comment (yaichenbaum) [inline builds/azure-pipelines-release.yml:35]: Can you revert this change?

File: src/Files.Uwp/Strings/en-US/Resources.resw [hunk 2]
```diff
@@ -2792,4 +2789,13 @@ We use App Center to track which settings are being used, find bugs, and fix cra
   <data name="ReviewFilesContent" xml:space="preserve">
     <value>Would you like to review Files?</value>
   </data>
+  <data name="SetAsBackgroundFlyout" xml:space="preserve">
+    <value>Set as background</value>
+  </data>
+  <data name="SetAsLockscreen" xml:space="preserve">
+    <value>Set as lockscreen background</value>
```
Comment (yaichenbaum) [inline src/Files.Uwp/Strings/en-US/Resources.resw:2796]:
```suggestion
    <value>Lockscreen</value>
```

File: src/Files.Uwp/Strings/en-US/Resources.resw [hunk 3]
```diff
@@ -2792,4 +2789,13 @@ We use App Center to track which settings are being used, find bugs, and fix cra
...
+  </data>
+  <data name="SetAsBackground" xml:space="preserve">
+    <value>Set as desktop background</value>
```
Comment (yaichenbaum) [inline src/Files.Uwp/Strings/en-US/Resources.resw:2799]:
```suggestion
    <value>Desktop</value>
```

File: builds/azure-pipelines-release.yml (hunk 1 above)
Comment (puppetsw) [reply builds/azure-pipelines-release.yml]: Should be fixed.

File: src/Files.Uwp/Strings/en-US/Resources.resw (hunk 3 above)
Comment (puppetsw) [reply src/Files.Uwp/Strings/en-US/Resources.resw:2799]: Should the Slideshow one be changed too?

Comment (yaichenbaum) [reply src/Files.Uwp/Strings/en-US/Resources.resw:2799]: I think slideshow should stay as is. Just verifying, when multiple images are selected, the only option shown is the slideshow button, is that correct?

Comment (puppetsw) [reply src/Files.Uwp/Strings/en-US/Resources.resw:2799]: No, it currently shows all options, but it will only apply the function to the first selected. I can change that though.

File: src/Files.Uwp/UserControls/InnerNavigationToolbar.xaml [hunk 4]
```diff
@@ -272,13 +274,42 @@
...
+                            <MenuFlyoutItem
+                                x:Name="SetAsBackgroundFlyoutItem"
+                                Command="{x:Bind ViewModel.SetAsBackgroundCommand, Mode=OneWay}"
+                                Text="{helpers:ResourceString Name=SetAsBackground}">
+                                <MenuFlyoutItem.Icon>
+                                    <FontIcon Glyph="&#xE91B;" />
+                                </MenuFlyoutItem.Icon>
+                            </MenuFlyoutItem>
+                            <MenuFlyoutItem
+                                x:Name="SetAsSlideshowFlyoutItem"
```
Comment (yaichenbaum) [inline src/Files.Uwp/UserControls/InnerNavigationToolbar.xaml:295]: Looking really good! Do you think this should be a dedicated toolbar button so that users don't have to open the flyout to access a single option?

Comment (puppetsw) [reply src/Files.Uwp/UserControls/InnerNavigationToolbar.xaml:295]: I like that idea If I'm understanding correctly. When you select one image it shows the Flyout for lockscreen and background, but when you select multiple it hides the flyout and replaces with single Slideshow button.

Comment (yaichenbaum) [reply src/Files.Uwp/UserControls/InnerNavigationToolbar.xaml:295]: That's exactly what I'm imagining 👀

Comment (yaichenbaum) [reply src/Files.Uwp/UserControls/InnerNavigationToolbar.xaml:295]: I like to call this the intuitive toolbar

Comment (puppetsw) [reply src/Files.Uwp/UserControls/InnerNavigationToolbar.xaml:295]: I'll make those changes.

**Comment by puppetsw** [comment]
recordingexample

I reverted some changes to the strings and added an extra for Lockscreen, as I think it makes more sense have individual ones for Lockscreen/Desktop instead of replacing SetAs_

File: src/Files.Uwp/Strings/en-US/Resources.resw [hunk 5]
```diff
@@ -2786,6 +2783,15 @@ We use App Center to track which settings are being used, find bugs, and fix cra
...
     <value>Would you like to review Files?</value>
   </data>
+  <data name="SetAsBackgroundFlyout" xml:space="preserve">
+    <value>Set as background</value>
+  </data>
+  <data name="SetAsLockscreen" xml:space="preserve">
+    <value>Set as lockscreen</value>
+  </data>
+  <data name="SetAsBackground" xml:space="preserve">
+    <value>Set as desktop background</value>
```
Comment (yaichenbaum) [inline src/Files.Uwp/Strings/en-US/Resources.resw:2793]: Are these still being used?

Comment (yaichenbaum) [reply src/Files.Uwp/Strings/en-US/Resources.resw:2793]: Ah, are these for the right click menu?

**Comment by yaichenbaum** [comment]
> recordingexample1 [ recordingexample1 ](https://user-images.githubusercontent.com/79826944/185817713-3fef703f-443e-416a-88dc-e50872622e65.gif) [ ](https://user-images.githubusercontent.com/79826944/185817713-3fef703f-443e-416a-88dc-e50872622e65.gif)
> 
> I reverted some changes to the strings and added an extra for Lockscreen, as I think it makes more sense have individual ones for Lockscreen/Desktop instead of replacing SetAs_

This is perfect!

File: src/Files.Uwp/Strings/en-US/Resources.resw (hunk 5 above)
Comment (puppetsw) [reply src/Files.Uwp/Strings/en-US/Resources.resw:2793]: I believe so

Comment (yaichenbaum) [reply src/Files.Uwp/Strings/en-US/Resources.resw:2793]: We should bring the right click menu in line with the toolbar but this is fine for now.

**Review by yaichenbaum — APPROVED** [review]
Excellent work! LGTM

File: src/Files.Uwp/Strings/en-US/Resources.resw (hunk 5 above)
Comment (puppetsw) [reply src/Files.Uwp/Strings/en-US/Resources.resw:2793]: It's next on the list.
